package.json

```diff
@@ -23,11 +23,9 @@
     "@hapi/joi": "^15.0.0",
     "D": "^1.0.0",
```

Comment:
What does this package do?

Reply:
this package was installed as a dependency to generate random data